pennylane/operation.py

```diff
@@ -420,8 +420,9 @@ class Operator(abc.ABC):
         params (tuple[tensor_like]): trainable parameters
         wires (Iterable[Any] or Any): Wire label(s) that the operator acts on.
             If not given, args[-1] is interpreted as wires.
-        do_queue (bool): indicates whether the operator should be
-            recorded when created in a tape context
+        do_queue (bool): indicates whether the operator should be recorded when created in
+            a tape context. This arguement is deprecated, instead of setting it to `False`
+            use `qml.QueuingManager.stop_recording()`.
```

Comment:
```suggestion
            a tape context. This arguement is deprecated, instead of setting it to ``False``
            use :meth:`~.QueuingManager.stop_recording`.
```
rst and Sphinx are annoying. Code like `False` needs double back ticks.  If we do something like `:meth:~.QueuingManager.stop_recording`, sphinx will insert a link to the relevant method. I think I got that syntax right at least. I might have done something wrong.


![Screenshot 2023-05-19 at 2 35 50 PM](https://github.com/PennyLaneAI/pennylane/assets/6364575/0221d640-a06e-412b-a2de-e4cce385b8f0)

Reply:
Oh, dear. Thanks. I changed all docstrings now: ef7e75f7809a8cbcb168a647c4d99002c90df60d